src/base/collection.js

```diff
@@ -749,18 +750,78 @@ CollectionBase.prototype._reset = function() {
 // Lodash methods that we want to implement on the Collection.
 // 90% of the core usefulness of Backbone Collections is actually implemented
 // right here:
-const methods = ['forEach', 'each', 'map', 'collect', 'reduce', 'foldl',
-  'inject', 'reduceRight', 'foldr', 'find', 'detect', 'filter', 'select',
-  'reject', 'every', 'all', 'some', 'any', 'include', 'contains', 'invoke',
-  'max', 'min', 'toArray', 'size', 'first', 'head', 'take', 'initial', 'rest',
-  'tail', 'drop', 'last', 'without', 'difference', 'indexOf', 'shuffle',
-  'lastIndexOf', 'isEmpty', 'chain'];
+
+const methods = {
```

Comment:
Right, I forgot 😄 

another thing I can do is remove as much as I can of the internal usage of lodash when it's applied to the other classes and switch it out for calling Lodash directly.

Example: `collection.invokeMap` => `_.invokeMap` as much as I can. That way, if we want to in the future drop all lodash methods from certain classes we can without any major code refactoring on the part of bookshelf.